week-1/exercises/paths.js

```diff
@@ -15,4 +15,11 @@ const base = filePath.slice(lastSlashIndex + 1);
 console.log(`The base part of ${filePath} is ${base}`);
 
 // Create a variable to store the dir part of the filePath variable
+const dir = filePath.slice(0, lastSlashIndex + 1);
+console.log(dir);
+
 // Create a variable to store the ext part of the variable
+const lastDotIndex = filePath.lastDotIndex(".");
```

Comment:
The code here looks good but unfortunally this line doesn't work. What error does it cause when you run it? Why does this error happen?

Reply:
yaaaa, write code in there : const lastDotIndex = filePath.lastIndexOf(".");

Reply:
Thanks for the replies, it's good to see that you understand. Try not to rush the work - it's so easy to miss little bits like this when you're in a hurry. Please try to attempt the rest of the work that you didn't manage to do yet :)